storage/store.go

```diff
@@ -1213,6 +1223,10 @@ func (s *Store) ExecuteCmd(ctx context.Context, args proto.Request) (proto.Respo
 			// This error needs to be converted appropriately so that
 			// clients will retry.
 			err = proto.NewRangeNotFoundError(rng.Desc().RangeID)
+		} else if iErr, ok := err.(*errWithIndex); ok {
```

Comment:
I know it doesn't give you the index, but for now it's unused. I didn't realize it was a test-only helper.